--- a/setup.py
+++ b/setup.py
@@ -6,12 +6,13 @@

 setup(
     name='XStatic-angular-ui-router',
+    version='0.3.1.2',
     summary="""angular-ui-router 0.3.1 (XStatic packaging standard)""",
     description=long_description,
     maintainer="Richard Jones",
     maintainer_email='r1chardj0n3s@gmail.com',
     use_scm_version=True,
-    setup_requires=['setuptools_scm', 'wheel'],
+#    setup_requires=['setuptools_scm', 'wheel'],
     packages=find_packages(),
     include_package_data=True
 )
