diff -ru oslo.vmware.orig/oslo_vmware/tests/test_service.py oslo.vmware/oslo_vmware/tests/test_service.py
--- oslo.vmware.orig/oslo_vmware/tests/test_service.py	2016-10-28 09:30:41.292421303 +0200
+++ oslo.vmware/oslo_vmware/tests/test_service.py	2016-10-28 09:21:19.992499965 +0200
@@ -386,8 +386,10 @@
         self.assertEqual('value2', cache.get('key2'))
         self.assertIsNone(cache.get('key3'))
 
-    @mock.patch('suds.reader.DocumentReader.download')
-    def test_shared_cache(self, mock_reader):
+    @mock.patch('suds.reader.DocumentReader._DocumentReader__fetch',
+                create=True)
+    @mock.patch('suds.reader.DocumentReader.download', create=True)
+    def test_shared_cache(self, mock_reader, mock_fetch):
         cache1 = service.Service().client.options.cache
         cache2 = service.Service().client.options.cache
         self.assertIs(cache1, cache2)
